src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/common/GetFormatDataCommand.cs

```diff
@@ -129,6 +130,7 @@ protected override void ProcessRecord()
 
             foreach (ViewDefinition definition in viewdefinitions)
             {
+                this.WriteVerbose(string.Format(CultureInfo.InvariantCulture, GetFormatDataStrings.ProcessRecord, definition.name));
```

Comment:
Output for user to read should be using the current culture.
```suggestion
                this.WriteVerbose(string.Format(CultureInfo.CurrentCulture, GetFormatDataStrings.ProcessRecord, definition.name));
```